libs/threading_base/include/hpx/threading_base/annotated_function.hpp

```diff
@@ -114,36 +113,40 @@ namespace hpx { namespace util
     };
 #endif
 
-    namespace detail
-    {
+    namespace detail {
         template <typename F>
         struct annotated_function
         {
             annotated_function() noexcept
               : name_(nullptr)
-            {}
+            {
+            }
 
             annotated_function(F const& f, char const* name)
-              : f_(f), name_(name)
-            {}
+              : f_(f)
+              , name_(name)
+            {
+            }
 
-            annotated_function(F && f, char const* name)
-              : f_(std::move(f)), name_(name)
-            {}
+            annotated_function(F&& f, char const* name)
+              : f_(std::move(f))
+              , name_(name)
+            {
+            }
 
         public:
-            template <typename ... Ts>
-            typename invoke_result<
-                typename util::decay_unwrap<F>::type, Ts...>::type
-            operator()(Ts && ... ts)
+            template <typename... Ts>
+            typename invoke_result<typename util::decay_unwrap<F>::type,
+                Ts...>::type
+            operator()(Ts&&... ts)
             {
                 return util::invoke(f_, std::forward<Ts>(ts)...);
             }
 
             template <typename Archive>
             void serialize(Archive& ar, unsigned int const /*version*/)
             {
-                ar & f_;
+                ar& f_;
```

Comment:
This is where clang-format always gets things wrong... Nothing we can do about this, I guess *sigh*.

Reply:
Yeah, this one might be worth a `// clang-format off`.